src/main/scala/ai/privado/languageEngine/default/processor/DefaultProcessor.scala

```diff
@@ -105,11 +102,6 @@ class DefaultProcessor(
 
     val xtocpg = withNewEmptyCpg(cpgOutputPath, cpgconfig: JavaConfig) { (cpg, config) =>
       {
-        println(
-          s"${TimeMetric.getNewTime()} - Base processing done in \t\t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
-        )
-        // Apply default overlays
-        X2Cpg.applyDefaultOverlays(cpg)
```

Comment:
I think applying default overlays is not required for default processor but cross check it once.

Reply:
These overlays are applied in code on the `main` branch.